src/runtime_src/xdp/profile/database/static_info/xclbin_info.cpp

```diff
@@ -21,26 +21,77 @@
 #include "xdp/profile/database/static_info/pl_constructs.h"
 #include "xdp/profile/database/static_info/xclbin_info.h"
 #include "xdp/profile/device/device_intf.h"
+#include "core/common/message.h"
 
 namespace xdp {
 
+  PLInfo& PLInfo::operator=(const PLInfo& other)
```

Comment:
Using "source"/"src" as the name of the source PLInfo, in place of "other" is more intuitive and readable.